style.css

```diff
@@ -338,7 +358,186 @@ nav a {
 }
 
 @media only screen and (min-width: 768px) {
+  header {
+    min-height: 160vh;
+    border-bottom-left-radius: 70px;
+  }
+
+  nav {
+    position: fixed;
+    top: 0;
+    background: #fff;
+    width: 98%;
+  }
+
+  .link img {
+    display: none;
+  }
+
+  .link ul {
+    display: block;
+  }
+
+  .link ul li {
+    display: inline-block;
+    padding: 0 10px;
+  }
+
+  .link ul li a {
+    color: #344563;
+    font-size: 15px;
+    font-weight: 500;
+    cursor: pointer;
+  }
+
   .background {
-    background: none;
+    min-height: 130vh;
+    background: url("./assets/Header\ bg.svg");
+    background-repeat: no-repeat;
+    background-position: center;
+    background-size: contain;
+    margin-top: 30px;
+  }
+
+  .intro {
+    padding: 100px 250px;
+  }
+
+  .text {
+    font-size: 40px;
+    letter-spacing: 0.37px;
+    margin-top: -50px;
+    line-height: 100px;
+  }
+
+  .intro-text {
+    font-size: 20px;
+    line-height: 28px;
+    margin-top: -10px;
+  }
+
+  .projects {
+    margin-top: 80px;
+    gap: 60px;
+  }
+
+  .project-row {
+    display: grid;
+    width: 1016px;
+    height: 496px;
+    padding: 0 38px;
+    grid-template-columns: repeat(2, auto);
+    grid-gap: 100px;
+  }
+
+  .project-row:nth-child(2n) .left {
+    order: 1;
+    margin-right: 80px;
+    margin-left: 0;
+    width: 440px;
+    height: 445px;
+    margin-top: 28px;
+  }
+
+  .left img {
+    width: 440px;
+    height: 445px;
+    margin-top: 28px;
+  }
+
+  .right h1 {
+    font-size: 40px;
+  }
+
+  .right .content {
+    font-size: 18px;
+  }
+
+  .right p {
+    font-size: 16px;
+  }
+
+  .about-me {
+    min-height: 140vh;
```

Comment:
- [ ] Kindly consider changing this to `min-height: 80vh;` as 140 makes your design different from the original design.
your design
![qw](https://user-images.githubusercontent.com/96683843/236251702-8081f7fe-6edd-4c13-ac48-8778ac217434.jpg)
figma design
![v](https://user-images.githubusercontent.com/96683843/236251796-6a83dc94-2467-4e02-9a57-86acce218c4e.jpg)